tests/test_formats.py

```diff
@@ -1179,3 +1686,243 @@ def test_fmt_image_path():
     formatter = FmtImage(encode=False, path="/a/b")
     res = formatter.to_html("c")
     assert 'src="/a/b/c"' in res
+
+
+# ------------------------------------------------------------------------------
+# Test `fmt_nanoplot()`
+# ------------------------------------------------------------------------------
+
+
+def _all_vals_nanoplot_output(nanoplot_str: List[str]):
+    import re
+
+    return all(bool(re.match("^<div><svg.*</svg></div>$", x)) for x in nanoplot_str)
+
+
+@pytest.mark.parametrize(
```

Comment:
It looks like many of the tests below...

* test across a wide range of inputs (10 rows in a DataFrame)
* only assert that the nanoplot exists with a div and svg tag

Please re-write so that it clear what each cases is testing, and their is a more specific assertion. For example, if reference line is "mean", does that mean the line is the average across rows? If so, in order to write these tests, you may need to reduce the number of input rows (e.g. to two), and test the exact value of the reference line in resulting nanoplots.

Reply:
If some of these parameters end up being tested more thoroughly in another place (e.g. `_generate_nanoplots`), then we should think about what parameters need to be tested at this level and how thoroughly (based on the underlying logic in the fmt_nanoplot function).